RISK_MATRIX.md

```diff
@@ -0,0 +1,61 @@
+# Risk consideration
+
+When merging a new feature of considerable size or modifying an existing one,
+consider adding a *Risk Matrix* section to your PR in collaboration with other
+developers on your team and the QA team.
+
+Below are some general themes to consider for the *Risk Matrix*:
```

Comment:
I'd phrase this as:

"Consider these areas of risk for the *Risk Matrix*:"

At the moment I think the list under general is a bit too long and some of the points are quite specific. Grouping them a bit more narrowly can help the reader orientate themselves and encourage them to the think of other points. For instance, we could do something like:

* General risks - Kibana-specific risks like spaces, plugin dependency, licensing
* Operational risks - points regarding environment in which code is running, clustered mode etc
* Performance risks - anything from server-side to client-side performance
* Security risks - this can remain as is IMO

Reply:
I like the idea of grouping in general, but we explicitly discussed with Petr and Peter to collect about "two dozen" of bullet, points, so I'm keeping it as is for now, and we can group later.